src/lib/locales/de.json

```diff
@@ -97,6 +97,10 @@
 		"qasim": {
 			"catchphrase": "☁boy 🤠",
 			"body": "Software Engineer. Begeistert von modernen Technologien und Sprachen. Immer auf der Suche nach dem geeignetsten Tool in der Schublade."
+		},
+		"philippe": {
+			"catchphrase": "Software Wizzard",
```

Comment:
`Wizard` is spelled with one `z`. Is the word `Wizzard` intentional?

Reply:
See above